src/Plugin.php

```diff
@@ -19,12 +19,24 @@
 use Psalm\SymfonyPsalmPlugin\Twig\CachedTemplatesTainter;
 use SimpleXMLElement;
 use Symfony\Component\HttpKernel\Kernel;
+use function array_merge;
 
 /**
  * @psalm-suppress UnusedClass
  */
 class Plugin implements PluginEntryPointInterface
 {
+    /**
+     * @return string[]
+     */
+    protected function getFormStubs(): array
+    {
+        $forms = glob(__DIR__ . '/Stubs/Form/*.stubphp') ?: [];
```

Comment:
I am not sure about this subfolder. We can still organize form stubs under a separate subfolder but not at this level since it is all about symfony versions and forms can be a subject of version change. I think it can be organized like this

* stubs
    * common
         * form
         * other folders
    * 4
        * form 
        * other folders
    * 5
    ...

After changing structure to this there is no need to change this class.

Reply:
I wanted to replicate structure that Symfony used, but can move them. Is it a problem to do that once everything else is done?
Just a note; there is no change between versions 4.4 (not sure about older) and 5.